pkg/ddevapp/settings_test.go

```diff
@@ -82,3 +93,174 @@ func TestWriteDrushConfig(t *testing.T) {
 	err = os.RemoveAll(dir)
 	assert.NoError(t, err)
 }
+
+// TestIncludeSettingsDdevInNewSettingsFile verifies that when no settings.php file exists,
+// a settings.php file is created that includes settings.ddev.php.
+func TestIncludeSettingsDdevInNewSettingsFile(t *testing.T) {
+	dir := testcommon.CreateTmpDir("")
+	err := os.MkdirAll(filepath.Join(dir, "sites/default"), 0777)
+	assert.NoError(t, err)
+
+	app, err := NewApp(dir, DefaultProviderName)
+	assert.NoError(t, err)
+
+	for appType, relativeSettingsLocations := range appTypeSettingsLocations {
+		app.Type = appType
+
+		relativeSettingsLocation := relativeSettingsLocations[0]
+		relativeSettingsDdevLocation := relativeSettingsLocations[1]
+		expectedSettingsLocation := filepath.Join(dir, relativeSettingsLocation)
+		expectedSettingsDdevLocation := filepath.Join(dir, relativeSettingsDdevLocation)
+
+		// Ensure that no settings.php exists
+		os.Remove(expectedSettingsLocation)
```

Comment:
These os.Remove() are  definitely going to get gometalinter complaints when you get here, err = and assert.NoError(), or _ =?

Reply:
Interesting, gometalinter hasn't complained about these.  I've updated to `_ =` anyway.